dotnet/src/Extensions/PromptTemplate.Handlebars/Helpers/KernelFunctionHelpers/KernelFunctionHelpers.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using HandlebarsDotNet;
+using HandlebarsDotNet.Compiler;
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+namespace Microsoft.SemanticKernel.PromptTemplate.Handlebars.Helpers;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Extension class to register Kernel functions as helpers.
+/// </summary>
+public static class KernelFunctionHelpers
```

Comment:
Is this intended to be extension methods as the comment indicates? If so, it should be named with an "Extensions" suffix, and the public method(s) should actually be extensions, e.g. it should be `Register(this IHandlebars handlebars, ...`.  Also, the name of the type starts with "KernelFunction" but this actually extends "IHandlebars"... should it be "Handlebarsextensions"?

Reply:
Opted for the former for now